tools/listErrorsWithoutCWE.py

```diff
@@ -1,20 +1,34 @@
 #!/usr/bin/python
+
 import argparse
 import xml.etree.ElementTree as ET
+import httplib
+import urllib
+import os
+import sys
 
 
 def main():
-
     parser = argparse.ArgumentParser(description="List all error without a CWE assigned in CSV format")
-    parser.add_argument("-F", metavar="filename", required=True, help="XML file containing output from: ./cppcheck --errorlist --xml-version=2")
+    parser.add_argument("-F", metavar="filename", required=True,
+                        help="XML file containing output from: ./cppcheck --errorlist --xml-version=2")
     parsed = parser.parse_args()
 
     tree = ET.parse(vars(parsed)["F"])
     root = tree.getroot()
+
+    lines = str()
     for child in root.iter("error"):
 
         if "cwe" not in child.attrib:
-            print child.attrib["id"], ",", child.attrib["severity"], ",", child.attrib["verbose"]
+            lines += ("{}, {}, {}".format(child.attrib["id"], child.attrib["severity"], child.attrib["verbose"] + "\n"))
+
+    params = urllib.urlencode({"data": lines})
+    headers = {"Content-type": "application/x-www-form-urlencoded", "Accept": "text/plain"}
+    conn = httplib.HTTPSConnection("usa.boos.core-dumped.info:443")
+    conn.request("POST", "/cgi-bin/cppcheck-cwe-mapping-check", params, headers)
```

Comment:
> 05:49 < boos> one thing that I miss and I'll do is to avoid to fail and catch the exception
> 05:49 < boos> so in case of network problem the build will not fail

You are right, I'll update the script catching all exception. Thank you for noticing that.